mmv1/products/firestore/api.yaml

```diff
@@ -17,9 +17,120 @@ versions:
   - !ruby/object:Api::Product::Version
     name: ga
     base_url: https://firestore.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://firestore.googleapis.com/v1/
 scopes:
   - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Google Cloud Firestore API
+    url: https://console.cloud.google.com/apis/library/firestore.googleapis.com
 objects:
+  - !ruby/object:Api::Resource
+    name: 'Database'
+    min_version: beta
+    base_url: 'projects/{{project}}/databases'
+    create_url: 'projects/{{project}}/databases?databaseId={{name}}'
```

Comment:
Should this use `database_id` instead? The name is then a separate output_only property that gives the full resource name if we want to be consistent with https://cloud.google.com/firestore/docs/reference/rest/v1/projects.databases#resource:-database